### ISSUE TITLE
require_once blocks UndefinedGlobalVariable warnings even when suppressed

### ISSUE THREAD

**Issue by olleharstedt** [vimeo/psalm#1768]
Reproduce:

1. Define a global string `$path` in psalm.xml
2. Analyse the following script:

```
require_once $path . 'some_file.php';
$asd = $qwe
```

The only notice is:

> INFO: UnresolvableInclude - test.php:14:1 - Cannot resolve the given expression to a file path

Would also expect:

> ERROR: UndefinedGlobalVariable - test.php:16:8 - Cannot find referenced variable $qwe in global scope

Not sure if this is by design?

Setting

    <UnresolvableInclude errorLevel="suppress" />

produces

> No errors found!

which could be considered a false negative.

**Comment by muglug** [comment]
`some_file.php` could contain code that sets `$qwe`, so Psalm plays it safe to avoid false positives.

**Comment by olleharstedt** [comment]
Thank you for your answer. Any way to change this behaviour?

**Comment by muglug** [comment]
Yes - you can PR a change that only does this if a config flag is set: https://github.com/vimeo/psalm/blob/010f911d22db94219166909d2f94b46da1896e61/src/Psalm/Internal/Analyzer/Statements/Expression/IncludeAnalyzer.php#L168-L170

**Comment by muglug** [comment]
`reportErrorsAfterUnknownInclude` or somesuch
